nf_core/pipeline-template/.gitpod.yml

```diff
@@ -3,9 +3,8 @@ tasks:
   - name: Update Nextflow and setup pre-commit
     command: |
       pre-commit install --install-hooks
-      nextflow self-update
+      nextflow self-update {%- if code_linters %}
 
 vscode:
   extensions:
-    #{%- if code_linters -%}
-    - nf-core.nf-core-extensionpack # https://github.com/nf-core/vscode-extensionpack{% endif %}
+    - nf-core.nf-core-extensionpack {%- endif -%}
```

Comment:
also not needed
```suggestion
    - nf-core.nf-core-extensionpack {% endif %}
```

Reply:
don't we want to keep the comment with the url?

Reply:
ah, sorry, yes